e2e/cypress/tests/integration/ad_ldap/saml_ldap_sync_remove_spec.ts

```diff
@@ -25,15 +25,15 @@ describe('AD / LDAP', () => {
     let testTeamId;
 
     before(() => {
-        cy.createLDAPUser().then((user) => {
+        (cy as any).createLDAPUser().then((user) => {
             samlLdapUser = user;
         });
 
         cy.shouldNotRunOnCloudEdition();
         cy.apiRequireLicenseForFeature('LDAP', 'SAML');
 
         // # Create new LDAP user
-        cy.createLDAPUser().then((user) => {
+        (cy as any).createLDAPUser().then((user) => {
```

Comment:
this is not an issue once you update to latest master

Reply:
updated